custom-connectors/AzureADPhoneMethods/apiDefinition.swagger.json

```diff
@@ -459,6 +474,433 @@
           }
         }
       }
+    },
+    "/beta/users/{userID}/authentication/phoneMethods/3179e48a-750b-4051-897c-87b9720928f7/enableSmsSignIn": {
+      "post": {
+        "summary": "Enable Sms SignIn",
+        "description": "Enable SMS Sign in",
+        "operationId": "EnableSmsSignin",
+        "parameters": [
+          {
+            "name": "userID",
+            "in": "path",
+            "type": "string",
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
@BakkerJan Please replace **ALL** the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.